### PR TITLE
updated nodemailer package for e-mailing

### PR DESCRIPTION
**Pull request by Rajsnehil**
Targets #549 

## Summary
Updated package nodemailer

## Test Plan

- [x] Wrote tests
- [x] Tested locally

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "@ramonak/react-progress-bar": "^3.1.0",
     "@sentry/react": "^6.16.1",
     "@sentry/tracing": "^6.16.1",
+    "@strapi/provider-email-nodemailer": "^4.6.0",
```
Comment (rathorevaibhav) [inline package.json:12]: why is strapi package needed in frontend?

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Rajsnehil) [reply package.json:12]: I had previously updated the nodemailer thinking this might be the issue. So, I pushed the changes.

Comment (rathorevaibhav) [reply package.json:12]: it doesn't look like you updated existing nodemailer. There was no nodemailer package previously installed and you have added a new package

Comment (Rajsnehil) [reply package.json:12]: @rathorevaibhav 
email-nodemailer was already installed with version 3.6 to which I updated it to 4.6

Comment (rathorevaibhav) [reply package.json:12]: @Rajsnehil I don't see nodemailer in the main branch package.json. Are you sure it was already installed?
[here's the link for package.json main branch](https://github.com/gitmedha/sis-frontend/blob/main/package.json)

**Pull request closed without merging**